tensorboard/plugins/histogram/metadata.py

```diff
@@ -31,6 +31,7 @@
 
 
 def create_summary_metadata(display_name, description):
+  """Create a `tf.SummaryMetadata` proto for histogram plugin data."""
```

Comment:
Use a `Return:` block? A bit redundant, but maybe consistent and the return value shows up in HTML-based docs (if we ever generate them).